Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
Traceback (most recent call last):
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/bot_LinkedIn.py", line 48, in <module>
    if check_publication() == 1 :
  File "/mnt/c/Users/hans9/OneDrive/Documents/Hans/Travail/LinkedIn/utils/ft_file_handle.py", line 38, in check_publication
    if date.today().strftime("%d-%m-%Y") in contents[2] :
IndexError: list index out of range
